src/constants.js

```diff
@@ -60,6 +64,7 @@ export const BONDS = {
   dai: "dai",
   frax: "frax",
   eth: "eth",
+  lusd: "lusd",
```

Comment:
Do we still use this object? we _shouldn't_

Reply:
removed